server/base/src/main/java/org/apache/accumulo/server/conf/util/ZooInfoViewer.java

```diff
@@ -328,6 +334,14 @@ private void printInstanceIds(final Map<String,InstanceId> instanceIdMap, PrintW
     writer.println();
   }
 
+  private void printResourceGroups(ServerContext context, PrintWriter writer) {
+    Set<String> groups = new TreeSet<>();
+    context.getZooCache().getChildren(Constants.ZRESOURCEGROUPS).forEach(groups::add);
+    String rgs = String.join(",", groups.toArray(new String[] {}));
```

Comment:
This looks like it prints comma-separated. The script is looking for space-separated.

Reply:
Fixed in 63fbabf